### ISSUE TITLE
Unstructured solid performance penalty when slice file output included

### ISSUE THREAD

**Issue by dvswenson** [firemodels/fds#8838]
I have just started testing some unstructured solids problems. One thing I noticed right away was that there was a big performance penalty when I added slices to the output. In the two attached models, the one without slices ran in 7 seconds and the one with three slices ran in 43 seconds.

Sincerely,
Daniel Swenson
Thunderhead Engineering
[user_guide_unstructured_solid.fds.txt](https://github.com/firemodels/fds/files/5367553/user_guide_unstructured_solid.fds.txt)
[user_guide_unstructured_solid_with_slices.fds.txt](https://github.com/firemodels/fds/files/5367555/user_guide_unstructured_solid_with_slices.fds.txt)

**Comment by marcosvanella** [comment]
Thank you Dan. We'll have a look.

**Comment by marcosvanella** [comment]
An update on this, Glenn noted that the 6x difference in run times happens in windows. Also, seems the difference is still there when swapping the geometry by an OBST of similar size.

**Comment by gforney** [comment]
Linux and osx difference closer to 1.5x. also difference proportional to
number of slice files output and number of time steps output. I wonder if
this is a file flushing issue.

**Comment by dvswenson** [comment]
Good idea to check with an OBST. So this is probably not a practical issue, since on larger FDS 6 models slices are not a noticeable cost in the solution.

Maybe close this case?

**Comment by marcosvanella** [comment]
Hi Dan, let's leave it open for now while we try to understand why we see such a difference in timings for IO between Windows and Unix systems.

**Comment by mcgratta** [comment]
<img width="1383" height="764" alt="Image" src="https://github.com/user-attachments/assets/ad572c60-1243-46d9-907f-278547737141" />

<img width="1390" height="848" alt="Image" src="https://github.com/user-attachments/assets/f8bd7f3e-16c9-4dff-bdbc-688ec5514256" />

The cases ran in 26 (without slices) and 22 (with) seconds on Linux with the latest source. On my Windows desktop, 43 s vs 39 s. Dan's original timing of 7 s back in 2020 must have been an error. I don't see a problem here with either OS.
